manifests/mounts.pp

```diff
@@ -0,0 +1,4 @@
+class nfs::mounts($mounts = {}){
```

Comment:
For the sake of homogeneity with the `nfs::exports` class, would you mind callingl this `$mountshash`?